### PR TITLE
Migrated Travis CI to Github Actions

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Migrated Travis CI to Github Actions
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | GA is :green_circle:  https://github.com/Progi1984/docker-internal-images/actions
| Possible impacts? | N/A

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - name: Set environment variable
+      run: echo "image=prestashop/docker-internal-images:${{ matrix.ps-versions }}" >> $GITHUB_ENV
+
+    - name: Docker Build
+      run: docker build --build-arg INSTALL_LANGUAGES=0 -t "${{ env.image }}" .
+      working-directory: ${{ matrix.ps-versions }}
+
+    - name: Docker Images
+      run: docker images
+      working-directory: ${{ matrix.ps-versions }}
```
Comment (Quetzacoalt91) [inline .github/workflows/docker.yml:26]: Is it needed or did you use it for debugging purpose?

Comment (Progi1984) [reply .github/workflows/docker.yml:26]: No it's because we need to be in the directory where the Dockerfile is.

Comment (Quetzacoalt91) [reply .github/workflows/docker.yml:26]: I mean this task, not only the `working-directory`

Comment (Progi1984) [reply .github/workflows/docker.yml]: Thanks. It's fixed.

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @Progi1984 and @Quetzacoalt91

**Pull request merged**